src/Types/ScriptLookups.purs

```diff
@@ -1058,6 +1060,25 @@ processConstraint mpsMap osMap = do
         -- constraint already.
         _cpsToTxBody <<< _outputs %= Array.(:) txOut
         _valueSpentBalancesOutputs <>= provideValue amount
+    MustPayToScriptInlineDatum vlh dat plutusValue -> do
+      -- TODO Check if vlh corresponds to v1 or v2?
+      networkId <- getNetworkId
+      let amount = fromPlutusValue plutusValue
+      runExceptT do
+        let
+          txOut = TransactionOutput
+            { address: validatorHashEnterpriseAddress networkId vlh
+            , amount
+            -- TODO: save correct datum and scriptRef, should be done in
+            -- Constraints API upgrade that follows Vasil
+            -- https://github.com/Plutonomicon/cardano-transaction-lib/issues/691
```

Comment:
Wrong issue?

Reply:
This issue has been used all over the place in script lookups, it either needs to be changed or reopened, I think it was accidentally closed from other changes